@juanse1904/day1.js

```diff
@@ -0,0 +1,17 @@
+var searchInsert = function(nums, target) {
+    const myIndex = nums.findIndex((i)=> i===target);
+    if(myIndex === -1){
+        let virtualIndex;
+        nums.forEach((item,index)=> {
+            if(item<target &&  nums[index+1]>target){
+                 virtualIndex = index + 1;
+            }
+         });
```

Comment:
@juanse1904 aprovechando que el array está ordenado, en lugar de iterar sobre cada elemento uno a uno para encontrar el índice, se puede usar binary search. La complejidad pasa de O(n) a O(log(n))

https://youtu.be/MFhxShGxHWc